docs/assets/jquery.js

```diff
@@ -0,0 +1,2 @@
+/*! jQuery v3.6.0 | (c) OpenJS Foundation and other contributors | jquery.org/license */
```

Comment:
Why add jQuery as a dependency? it doesn't add much nowadays

Reply:
Well you could blame my school for that, we were forced to use it. I used just plain javascript and got a 0. (And no it wasn't like years ago ...) Now after using it a lot I find it clearer to read for the troubleshooter.